sass/components/_chart-title.scss

```diff
@@ -1,122 +1,123 @@
 .cq-chart-title {
+    display: inline-block;
     top: 1em;
     z-index: 41;
     margin: 0.2em 0.8em;
 
-    &.stxMenuActive {
```

Comment:
most of the styling here is only used in chart controls, so I moved them over there instead. This prevents chart control styles from bleeding into the chart title